src/test/java/com/amazon/opendistroforelasticsearch/security/securityconf/impl/v6/ConfigV6Test.java

```diff
@@ -79,7 +79,7 @@ public ConfigV6Test(boolean omitDefaults) {
     }
 
     @Test
-    public void testKibana() throws Exception {
+    public void testDashboards() throws Exception {
         ConfigV6.Kibana kibana;
```

Comment:
Should the class `ConfigV6.Kibana` be renamed to `ConfigV6.OpenSearchDashboards`? Also the variable name should be renamed.

Reply:
As we discussed, we'll take care of this change in the reset of the migration tasks.